content/chapters/compute/lecture/slides.mdpp

```diff
@@ -636,13 +633,7 @@ student@os:~$ ps -e -o pid,ni,comm | grep 5753
 
 ### Synchronization Mistake - TOCTOU
 
-```c
-if (lock = 0) {
-	lock = 1;
-    /* critical section */
-    lock = 0;
-}
-```
+`Time-Of-Check to Time-Of-Use`
```

Comment:
```suggestion
Time-Of-Check to Time-Of-Use
```
Why remove the code? It's useful to have it on the slide for printing / discussion purposes.

Reply:
The code is repeated in the gif `./media/race-condition-toctou-generated.gif`.

Reply:
I know. But if you print the slides, I'm not sure what happens to the gif. The code on the slide remains.

Reply:
I will leave it as it is (with the code) for now.